src/backend/src/routes/users.routes.ts

```diff
@@ -55,4 +55,14 @@ userRouter.post(
   UsersController.getManyUserTasks
 );
 
+userRouter.post(
```

Comment:
Dont we have a notifications router already? I feel like this should be under that